README.md

```diff
@@ -184,3 +191,85 @@ Required dependencies:
 _The following is to be done in the root directory_
 
 Hooks are installed with when running `make install_git_hooks`, installing both the python pre-commit hooks as well as a commit-msg hook by cocogitto
+
+## OpenShift information
+
+### General information
+
+- [The Private Cloud as a Service Platform Technical Documentation](https://docs.developer.gov.bc.ca/)
+
+### Deploying the project
+
+To deploy the project into a a new namespace or to deploy another instance of the project into an existing namespace GitHub Environments along with Helm and GitHub Actions is used. The following steps can be used to as a reference to deploy:
```

Comment:
I believe we need to first manually deploy with Helm to generate our deployer service account, then we can add the environment variables and deploy through GitHub. 

There is a default `deployer` service account set up in the namespaces but I am pretty sure it doesn't have all the permissions needed which is why we generate our own eg `ccbc-deployer`.

Reply:
Added some info on that.